javascript/phone-number.js

```diff
@@ -0,0 +1,27 @@
+module.exports = PhoneNumber;
+
+function PhoneNumber (num) {
+  num = num.replace(/[(]|[)]|[-]|[\s]|[.]/g, '');
```

Comment:
An alternative, more terse approach to consider would be to utilize the `\D` character to replace all non-digits.